core-bundle/assets/controllers/toggle-fieldset-controller.js

```diff
@@ -129,4 +138,12 @@ export default class extends Controller {
             })
         });
     }
+
+    setAriaAttributes(state) {
```

Comment:
```suggestion
    setAriaExpanded(state) {
```

(would need adjusting the callees as well.